README.md

```diff
@@ -36,7 +36,7 @@ By far the most common case will be installing the [eslint-plugin-typescript](ht
 
 The following additional configuration options are available by specifying them in [`parserOptions`](https://eslint.org/docs/user-guide/configuring#specifying-parser-options) in your ESLint configuration file.
 
-**`jsx`** - default `false`. Enable parsing JSX when `true`. More details can be found [here](https://www.typescriptlang.org/docs/handbook/jsx.html).
+**`jsx`** - default `false`. It's `true` on `*.tsx` files automatically. Enable parsing JSX when `true`. More details can be found [here](https://www.typescriptlang.org/docs/handbook/jsx.html).
```

Comment:
I wonder if we can make this clearer by explaining that the parser will auto detect this and this config option is only for when a file name is not provided.

Reply:
I updated it.